### PR TITLE
Conditional nav buttons

### PR DESCRIPTION
**Pull request by blackmacaroon**
# Description

adding ternary operator to navbar buttons show appropriate buttons based on user roles

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x]  I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -1,16 +1,21 @@
 import React from "react"
+import { connect } from "react-redux"
 import AppBar from "@material-ui/core/AppBar"
 import Toolbar from "@material-ui/core/Toolbar"
 import Button from "@material-ui/core/Button"
 import { AuthButton } from "./Auth"
-// import Typography from '@material-ui/core/Typography';
```
Comment (lappjeff) [inline src/components/NavBar.js]: Lets remove this unused import

File: src/components/NavBar.js [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
 import redLambdaLogo from "../assets/Lambda_Logo.png"
 
+
 // I've imported Material UI packages above,
 // and implemented them in a functional component below
 
 const NavBar = props => {
+  const { user } = props
+  console.log("user", user)
+  // console.log('user role', user.role)
```
Comment (lappjeff) [inline src/components/NavBar.js]: Make sure to remove your console logs before you take it off draft stage!

File: src/components/NavBar.js [hunk 3]
```diff
@@ -34,12 +39,26 @@ const NavBar = props => {
...
+                  ) : (
+                    <Button
+                      className="projectFormButton"
+                      onClick={() => props.history.push("/project-form")}
+                    >
+                      Submit Project
+                    </Button>
+                  
+                )
+              ) : null }
```
Comment (lappjeff) [inline src/components/NavBar.js]: Is this last `: null` line doing anything?

Comment (blackmacaroon) [reply src/components/NavBar.js]: yeah, it's not rendering either when you're not logged in. so they cant see pending and they cant submit apps

File: src/components/NavBar.js (hunk 2 above)
Comment (blackmacaroon) [reply src/components/NavBar.js]: thank you! we're still trying to figure this out, i'll remove them before we're ready for review

File: src/components/NavBar.js (hunk 1 above)
Comment (blackmacaroon) [reply src/components/NavBar.js]: yeet!

**Comment by blackmacaroon** [comment]
looks like we were all trying to clear the console 😂

File: src/components/NavBar.js [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
 import { MemoryRouter as Router, withRouter } from "react-router-dom"
 import redLambdaLogo from "../assets/Lambda_Logo.png"
 
+
 // I've imported Material UI packages above,
 // and implemented them in a functional component below
 
 const NavBar = props => {
+  const { user } = props
+  console.log("user", user)
```
Comment (lappjeff) [inline src/components/NavBar.js]: Console log on line 16 here 💭

**Pull request merged**